grapheneapi/graphenews.py

```diff
@@ -179,16 +181,21 @@ def run_forever(self) :
                     coro = loop.create_connection(self.factory, self.host,
                                                   self.port, ssl=context)
                 else :
+
                     coro = loop.create_connection(self.factory, self.host,
                                                   self.port, ssl=self.ssl)
-
                 loop.run_until_complete(coro)
                 loop.run_forever()
             except KeyboardInterrupt:
                 break
 
-            print("Trying to re-connect in 10 seconds!")
-            time.sleep(10)
-
+            try:
+                print("Multiple witnesses configured, trying another witness.")
```

Comment:
Please don't use `print` in the library. Instead, we should start using `logging` library wide. The message above would probably fit best into

    logging.info('...')